src/test/javascript/spec/integration/code-editor/code-editor-container.integration.spec.ts

```diff
@@ -204,11 +190,11 @@ describe('CodeEditorContainerIntegration', () => {
         const isCleanSubject = new Subject();
```

Comment:
These definitions could be inlined

Reply:
As discussed on Slack. I adapted some but for some it is not possible.